docs/reference/settings/security-settings.asciidoc

```diff
@@ -2320,19 +2321,18 @@ for every request, set to `0` to disable the JWT cache. Defaults to `20m`.
 // tag::jwt-pkc-jwkset-path-tag[]
 `pkc_jwkset_path` {ess-icon}::
 (<<static-cluster-setting,Static>>)
-The file path or URL to a JSON Web Key Set (JWKS) with the public key material that
-the JWT Realm uses for verifying token signatures. If a path is provided,
-then it is resolved relative to the {es} configuration directory. In {ecloud},
-use an absolute path starting with `/app/config/`.
+The file name or URL to a JSON Web Key Set (JWKS) with the public key material that
+the JWT Realm uses for verifying token signatures. A value is considered a file name
+if it does not begin with `https://`. The file name
```

Comment:
Strictly speaking JWT realm [does not care about the `://` part](https://github.com/elastic/elasticsearch/blob/735329e1108a085ef36734872244e925cc196a01/x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/jwt/JwtUtil.java#L164C22-L164C22). So this could be:
```suggestion
if it does not begin with `https` or `http`. The file name
```